### DISCUSSION TITLE
Talk App can't be enabled or removed !

### DISCUSSION THREAD

**Discussion by The13east** [nextcloud/all-in-one#2989] [Questions] [answered]
## Steps to reproduce
1.installed NextCloudAIO container on docker 
2.Everything is working and Talk is installed and running
3.There is no Talk in the Dashboard and in the setting It's disabled and when you try to enable it this error will pop out : An exception occurred while executing a query: SQLSTATE[42P07]: Duplicate table: 7 ERROR: relation "ts_session" already exists
removing Talk in order to reinstall it wont work and after reinstalling it it'll go to the same error and same behavior.

Talk : 17.0.1
MySql -V : mysql  Ver 8.0.33-0ubuntu0.22.04.2 for Linux on aarch64 ((Ubuntu))
PHP 8.1
 I'm a super noob so pls Walk me through pretty much everything

**Comment by szaimen** [comment]
This cannot be AIO as AIO does not use mysql.

**Comment by The13east** [comment]
> This cannot be AIO as AIO does not use mysql.

yeah nvm my dumbass is checking the OS not the container ! 
so I cant use talk while using nextcloud AIO ? or is there a solution ?

**Comment by szaimen** [comment]
Sorry, I am confused now. Did you install Nextcloud AIO and run into this issue or did you only install the talk container from it?

**Comment by szaimen** [comment]
Cc @nickvergessen is it possible that this error only happens with posgresql?

**Comment by szaimen** [comment]
Edit: i see the uniqueindex was added 3 years ago so this should usually not come up suddenly so no clue why it does.

Sorry for pinging you!

**Comment by szaimen** [comment]
@The13east may I ask how much RAM does your server has and which optional addons do you have enabled via the AIO interface?

**Comment by szaimen** [comment, marked as answer]
Is getting handled in https://help.nextcloud.com/t/error-with-talk-in-new-installation/166432/2
